securedrop/request_that_secures_file_uploads.py

```diff
@@ -1,14 +1,16 @@
 from io import BytesIO
+from typing import Any
 
 from flask import wrappers
+from werkzeug.formparser import FormDataParser
 
 from secure_tempfile import SecureTemporaryFile
 
 
 class RequestThatSecuresFileUploads(wrappers.Request):
 
-    def _secure_file_stream(self, total_content_length, content_type,
-                            filename=None, content_length=None):
+    def _secure_file_stream(self, total_content_length: int, content_type: Any,
```

Comment:
If I've followed the Flask and Werkzeug [source](https://github.com/pallets/werkzeug/blob/9444a66dec9c2ee1bacb5c9846669afcca692875/src/werkzeug/wrappers/base_request.py#L258) correctly, shouldn't `content_type` and `filename` be `Optional[str]`, and `content_length` be `Optional[int]`?

Reply:
Correct - I couldn't track it down when I looked at this, but I pushed the updated types just now.